db/migrate/20170905222833_add_default_value_to_availability.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultValueToAvailability < ActiveRecord::Migration[5.1]
+  def change
+    change_column_default :users, :availability, false
```

Comment:
Why are you using a separate migration for this? I'd suggest to just do it in the migration where the column is added.